src/controllers/accommodation.js

```diff
@@ -16,8 +18,11 @@ export default class accomodationFacility {
 
   static async getSingleAccommodation(req, res) {
```

Comment:
hey @EugeneMunya , 
Add
```
try{  }
catch (error) {
      return handleError(res, error);
    }
```
to catch this error:
![image](https://user-images.githubusercontent.com/30593186/77062238-27d5bb80-69e4-11ea-87e5-a1dd3832b133.png)

![image](https://user-images.githubusercontent.com/30593186/77062269-3c19b880-69e4-11ea-8e64-853f863c4e38.png)

Reply:
I am working on it right now

Reply:
@EugeneMunya , can you make this error more descriptive???

![image](https://user-images.githubusercontent.com/30593186/77068478-5d33d680-69ef-11ea-928d-776fe317f2c4.png)

Reply:
I did you can check for now